src/drivers/imu/invensense/mpu9250/MPU9250.hpp

```diff
@@ -80,7 +80,7 @@ class MPU9250 : public device::SPI, public I2CSPIDriver<MPU9250>
 	static constexpr float GYRO_RATE{1e6f / FIFO_SAMPLE_DT}; // 8 kHz gyro
 	static constexpr float ACCEL_RATE{GYRO_RATE / 2.f};      // 4 kHz accel
 
-	static constexpr uint32_t FIFO_MAX_SAMPLES{math::min(FIFO::SIZE / sizeof(FIFO::DATA), sizeof(PX4Gyroscope::FIFOSample::x) / sizeof(PX4Gyroscope::FIFOSample::x[0]))};
+	static constexpr uint32_t FIFO_MAX_SAMPLES{math::min(FIFO::SIZE / sizeof(FIFO::DATA), sizeof(sensor_gyro_fifo_s::x) / sizeof(sensor_gyro_fifo_s::x[0]))};
```

Comment:
This makes it a bit more robust:
```suggestion
	static constexpr uint32_t FIFO_MAX_SAMPLES{math::min(math::min(FIFO::SIZE / sizeof(FIFO::DATA), sizeof(sensor_gyro_fifo_s::x) / sizeof(sensor_gyro_fifo_s::x[0])), sizeof(sensor_accel_fifo_s::x) / sizeof(sensor_accel_fifo_s::x[0])*(GYRO_RATE/ACCEL_RATE))};
```

Reply:
Ah yes, cheating on the accel size. Good idea.